CHANGELOG.md

```diff
@@ -1,5 +1,112 @@
 # Release Notes
 
+## Version 1.19.0-beta.0 - 2021-04-05
+
+Features:
+
+* add minikube image rm command [#10924](https://github.com/kubernetes/minikube/pull/10924)
+* create registry credentials when gcp-auth addon is enabled [#10853](https://github.com/kubernetes/minikube/pull/10853)
```

Comment:
lets add colons liek this

GCP-Auth addon: Add support for GCR creds

Reply:
done